packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -182,3 +191,18 @@ export const localVideoCameraCycleButtonStyles: IButtonStyles = {
     background: 'transparent'
   }
 };
+
+/**
+ * Styles for the local video tile modal when it is focused, will cause keyboard move icon to appear over video
+ * @private
+ */
+export const localVideoModalStyles: IModalStyles = {
+  keyboardMoveIconContainer: {
+    zIndex: 6 // zIndex to set the keyboard movement Icon above the other layers in the video tile.
+  },
+  root: undefined,
```

Comment:
Are you able to avoid assigning all these undefineds by changing the return type from `IModalStyles` to `Partial<IModalStyles>`

Reply:
yeah without the undefined sections of it the linter complains, we could try disabling the check for it?

Reply:
even with wrapping the return type in `Partial<>`? 
```ts
export const localVideoModalStyles: Partial<IModalStyles> = {
```

Where does it complain?

Reply:
Otherwise try this type:
`IStyleFunctionOrObject<IModalStyleProps, IModalStyles>`